document-store/src/main/java/org/hypertrace/core/documentstore/mongo/parser/MongoIdentifierPrefixingSelectTypeExpressionParser.java

```diff
@@ -5,11 +5,11 @@
 import java.util.Optional;
 import org.hypertrace.core.documentstore.expression.impl.IdentifierExpression;
 
-final class MongoIdentifierPrefixingSelectingExpressionParser
-    extends MongoSelectingExpressionParser {
+final class MongoIdentifierPrefixingSelectTypeExpressionParser
```

Comment:
I would keep it `MongoIdentifierPrefixingSelectingExpressionParser` or simply `MongoIdentifierPrefixingExpressionParser` or `MongoIdentifierPrefixingSelectionParser`

Reply:
yeah right, I also prefer `MongoIdentifierPrefixingSelectionParser`